docs/guides/libraries-usage.md

```diff
@@ -0,0 +1,195 @@
+# Libraries usage
+
+You can use libraries across the app using two strategies:
+
+1. **Eager Load**: Loads the lib when the app is bootstrapped.
+2. **Lazy Loading**: Loads only when the app required it.
+
+### Eager Loading
+
+A good example was the usage of the `ui` library created previously. The lib was called directly on the `app.module.ts`
```

Comment:
Guides are meant to be read independently. So the reader may not not what ui library you are referring to.

Reply:
You right. The changes were made.